charts/firefly-iii-stack/Chart.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: v2
 name: firefly-iii-stack
-version: 0.5.2
+version: 0.6.0
 description: Installs Firefly III stack (db, app, importer)
 type: application
 dependencies:
 - name: firefly-db
-  version: 0.0.4
+  version: 0.2.1
```

Comment:
This version bump won't work as the new `firefly-db` chart won't be released at the time that the workflow wants to build the `firefly-iii-stack` chart.

For now, please drop the changes to the `firefly-iii-stack` chart from this PR and open a seperate PR for them. This will be automated in the future. 

Sorry for not having documented this, I thought I did so.